Change favicon

DEJENERUTA opened the pull request.

DEJENERUTA committed:
change react favicon to heal health logo

DEJENERUTA committed:
Add space between 'at' and email address in Join component (line 24)

bence-toth commented:
I believe the following should be updated as well:
- `logo192.png`
- `logo512.png`

I'm also wondering if we should ditch the `favicon.ico` and just use the `logo192.png` version instead.

Finally, why is the favicon black and white? Shouldn't we use transparent + Persian green (the logo color)?

DEJENERUTA committed:
Update logo files and favicon

DEJENERUTA commented:
This commit updates the following files as requested:
- Updated logo192.png and logo512.png to the latest version.
- Replaced favicon.ico with logo192.png for consistency.

DEJENERUTA committed:
delete favicon.ico, update index.html and manifest.json

DEJENERUTA committed:
fix Lint

bence-toth commented on public/manifest.json:
```diff
@@ -2,11 +2,6 @@
-    },
     {
       "src": "logo192.png",
       "type": "image/png",
```
`"sizes"` here should be `192x192 64x64 32x32 24x24 16x16`

bence-toth commented on public/index.html:
```diff
@@ -2,18 +2,14 @@
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logo192.png" />
```
```suggestion
    <link rel="icon" type="image/png" href="%PUBLIC_URL%/logo192.png" />
```

DEJENERUTA committed:
minor change

DEJENERUTA committed:
change logo512.png size to 64x64

DEJENERUTA committed:
fix logo192.png size

bence-toth committed:
Updated favicon link in index.html

bence-toth committed:
Updated Join.tsx file (white space)
